integration-tests/tests/src/tests/sync/flexible.ts

```diff
@@ -411,6 +411,15 @@ describe("Flexible sync", function () {
             closeRealm(realm2);
           });
 
+          it("sets the correct path from the configuration", async function () {
+            const realm = await Realm.open({
+              path: "custom_path.realm",
+              ...getSuccessConfig(this.user, {}),
+            });
+
+            expect(realm.path).to.contain("custom_path.realm");
```

Comment:
For an exact string to match, see e.g. the [local realm test](https://github.com/realm/realm-js/blob/1b24533db033d1cae13f2da62c3cb081cddb44e6/integration-tests/tests/src/tests/realm-constructor.ts#L52) 👍

Reply:
I believe it incorporates `user.id` if I'm not misremembering.

Reply:
Got it now!